source/sass/views/home.scss

```diff
@@ -114,3 +114,64 @@
     }
   }
 }
+
+/**
+* Partner section
+*/
+.partner-section {
+  background-size: cover;
+  background-position: center center;
+  position: relative;
+  z-index: 10;
+  padding-top: 80px;
+  padding-bottom: 80px;
+}
+
+.partner-section::before {
+  content: '';
+  position: absolute;
+  z-index: 0;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background-color: rgba(0, 0, 0, 0.6);
+}
+
+.partner-section-text {
+  font-family: $font-family-sans-serif;
+  font-size: 20px;
+  font-weight: 300;
+  line-height: 1.5;
+  max-width: 450px;
+}
+
+.partner-section-button {
+  font-family: $font-family-sans-serif;
+  font-size: 18px;
+  font-weight: bold;
+  line-height: 1.5;
+  display: inline-block;
+}
+
+.partner-section-logos {
+  position: relative;
+  z-index: 1;
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.partner-section-logo {
+  margin-right: 48px;
+  margin-top: 75px;
+}
+
+@media(max-width: $bp-md) {
+  .partner-section-logos {
+    display: none;
```

Comment:
this can be replaced by Bootstrap's class: https://getbootstrap.com/docs/4.0/utilities/display/